openslides/core/static/js/core/start.js

```diff
@@ -30,6 +30,40 @@ angular.module('OpenSlidesApp.core.start', [])
             }
         });
     }
+])
+
+.run([
+    '$rootScope',
+    '$state',
+    'operator',
+    'User',
+    'Group',
+    'mainMenu',
+    function ($rootScope, $state, operator, User, Group, mainMenu) {
+        var permissionChangeCallback = function () {
+            operator.reloadPerms();
+            mainMenu.updateMainMenu();
+            var basePerm = $state.current.data.basePerm;
+            $rootScope.baseViewPermissionsGranted = basePerm ?
+                operator.hasPerms(basePerm) : true;
+        };
+
+        $rootScope.$watch(function () {
+            return Group.lastModified();
+        }, function () {
+            if (Group.getAll().length) {
+                permissionChangeCallback();
+            }
+        });
+
+        $rootScope.$watch(function () {
+            return operator.user ? User.lastModified(operator.user.id) : true;
```

Comment:
I think we need the setUser method, but the permissions does not have to be set in this method because the watcher does this now.